eng/DotNetBuild.props

```diff
@@ -8,32 +8,23 @@
     <SourceBuildTargetFrameworkFilter>netstandard2.0%3bnetstandard2.1%3bnetcoreapp2.1%3bnetcoreapp3.1%3bnet5.0%3bnet6.0%3bnet7.0%3bnet8.0%3bnet9.0</SourceBuildTargetFrameworkFilter>
   </PropertyGroup>
 
-  <!-- Only run this target in source-build only mode. -->
-  <Target Name="PrepareGlobalJsonForSourceBuild"
```

Comment:
This shouldn't be required any longer, at least not in the VMR. We mutate the global.json to remove runtimes.